package.json

```diff
@@ -20,7 +20,8 @@
     "release": "node scripts/release.js",
     "release:publish": "node scripts/publish.js",
     "release:clean": "node scripts/cleanup.js",
-    "update-useragent": "node projects/auth0-angular/scripts/update-useragent.js"
+    "update-useragent": "node projects/auth0-angular/scripts/update-useragent.js",
+    "server:api": "node api-server.js"
```

Comment:
Do we want to have this server run in conjunction alongside the start command? Or is the expectation that we would just start it manually whenever we need it?